scripts/update-image.py

```diff
@@ -299,9 +300,9 @@ def run(self):
 			# Determine root uuid manually for mountpoint
 			diskdev, partdev = dev_for_mountpoint(self.mountpoint)
 
-			if os.isdir('/dev/disk/by-uuid'):
+			if os.path.isdir('/dev/disk/by-uuid'):
```

Comment:
`import os.path as path` is a common convention.

Reply:
I can replace all instances of `os.path` -> `path`, if that's what you're suggesting?